symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkSSLConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.symphony.bdk.core.config.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BdkSSLConfig {
```

Comment:
I would keep acronyms as camel case to avoid wondering witch case to use (for if you have an HTTPSSL class it would be weird, with HttpSsl we are consistent no matter what)